storage-sample.go

```diff
@@ -1,11 +1,11 @@
 /*
-Copyright 2013 Google Inc.
+Copyright 2015 Google Inc.
```

Comment:
Please do not change Copyright dates.  I'm told that it is important to leave the original date.